service/utils/schema.sql

```diff
@@ -1,25 +1,29 @@
-CREATE TABLE users (
-  userId INT NOT NULL,
-  username VARCHAR NOT NULL,
-  firstName VARCHAR NOT NULL,
-  lastName VARCHAR NOT NULL,
-  email VARCHAR NOT NULL UNIQUE,
-  age: INT,
-  snack: VARCHAR,
-  password_hash VARCHAR,
-  followers INT NOT NULL DEFAULT 0,
-  following INT NOT NULL DEFAULT 0,
+DROP TABLE IF EXISTS friendship;
+DROP TABLE IF EXISTS users;
 
-  PRIMARY KEY (username)
+CREATE TABLE users (
+    username varchar not null,
+    first_name varchar,
+    last_name varchar,
+    email varchar not null unique,
+    age int,
+    snack varchar,
+    follower_count int not null default 0,
+    following_count int not null default 0,
+    thumbnail bytea,
+    oauth boolean default false,
+    password_hash varchar not null
 );
 
-CREATE TABLE follow (
-  follower VARCHAR NOT NULL,
-  following VARCHAR NOT NULL,
+CREATE TABLE friendship (
+  friendshipId SERIAL,
```

Comment:
Not familiar with SERIAL, will need to look that up

Reply:
From what I've seen its auto increment id for Postgre

Reply:
👍